packages/adapter-pg/README.md

```diff
@@ -1,3 +1,65 @@
 # @prisma/adapter-pg
 
-**INTERNAL PACKAGE, DO NOT USE**
+This package contains the driver adapter for Prisma ORM that enables usage of the [`node-postgres`](https://node-postgres.com/) (`pg`) database driver for PostgreSQL. You can learn more in the [documentation](https://www.prisma.io/docs/orm/overview/databases/postgresql#using-the-node-postgres-driver).
```

Comment:
```suggestion
This package contains the driver adapter for Prisma ORM that enables usage of the [`node-postgres`](https://node-postgres.com/) (`pg`) database driver for PostgreSQL. You can learn more in the [documentation](https://pris.ly/d/adapter-pg).
```

Let's make this a shortlink? See suggestion (created on Dub)
I think it will be a good "long term investment"